api/src/main/java/uk/gov/hmcts/payment/api/controllers/OrderController.java

```diff
@@ -0,0 +1,254 @@
+package uk.gov.hmcts.payment.api.controllers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.swagger.annotations.*;
+import org.apache.commons.validator.routines.checkdigit.CheckDigitException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.bind.annotation.*;
+import uk.gov.hmcts.payment.api.domain.model.OrderPaymentBo;
+import uk.gov.hmcts.payment.api.domain.service.OrderDomainService;
+import uk.gov.hmcts.payment.api.dto.mapper.CreditAccountDtoMapper;
+import uk.gov.hmcts.payment.api.dto.order.OrderDto;
+import uk.gov.hmcts.payment.api.dto.order.OrderPaymentDto;
+import uk.gov.hmcts.payment.api.exception.AccountNotFoundException;
+import uk.gov.hmcts.payment.api.exception.AccountServiceUnavailableException;
+import uk.gov.hmcts.payment.api.exception.LiberataServiceTimeoutException;
+import uk.gov.hmcts.payment.api.exceptions.OrderReferenceNotFoundException;
+import uk.gov.hmcts.payment.api.model.*;
+import uk.gov.hmcts.payment.api.v1.model.exceptions.*;
+
+import javax.validation.Valid;
+import java.math.BigDecimal;
+import java.util.Optional;
+import java.util.function.Function;
+
+@RestController
+@Api(tags = {"Order"})
+@SwaggerDefinition(tags = {@Tag(name = "OrderController", description = "Order REST API")})
+public class OrderController {
+
+    private static final Logger LOG = LoggerFactory.getLogger(OrderController.class);
+    private static final String FAILED = "failed";
+
+    @Autowired
+    private OrderDomainService orderDomainService;
+
+    @Autowired
+    private IdempotencyKeysRepository idempotencyKeysRepository;
+
+    @Autowired
+    private Payment2Repository payment2Repository;
```

Comment:
it seems like we are not using this in this class. Please remove if that is the case

Reply:
Naming also is inappropriate. Please choose a better naming convention

Reply:
Removed the class